### PR TITLE
author info

### PR DESCRIPTION
**Pull request by loriab**
## Description
I was browsing some MolSSI-tweeted videos and learned about the `codemeta.json` movement. Didn't sound like a bad idea, so here's a start, reformatted from the 1.1 paper. It potentially needs more authors from psi1-psi3 era.

## Questions
- [ ] Do we want this?

## Checklist
- [x] I did run it against the schema checker https://github.com/codemeta/codemeta/blob/master/codemeta.jsonld

## Status
- [x] Ready for review
- [ ] Ready for merge

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    "downloadUrl": "https://github.com/caltechlibrary/datatools/archive/v0.0.24.zip",
+    "keywords": [
+        "csv",
+        "json",
+        "xlsx",
+        "golang",
+        "bash"
+    ],
+    "maintainer": "https://orcid.org/0000-0003-0900-6903",
+    "programmingLanguage": [
```
Comment (dgasmith) [inline codemeta.json]: Add `C` and maybe Fortran?

Comment (loriab) [reply codemeta.json]: will do. and ping anyone nearby who doesn't have their orcid filled in

**Comment by loriab** [comment]
thanks, @ashutoshvt, got it.

@robparrish, @edeprince3, @bozkaya, @ssh2, @ryanmrichard, @jgonthier , @amjames , @harls , @msaitow, @xwang862 , @bennybp , @PrakashVermaQTP , @PeterKraus , @hokru, @schiebermc , @robashaw, send your ORCID if you want it recorded.

**Comment by ssh2** [comment]
My ORCID is: 0000-0003-2637-4134. Thank you, Lori!

**Comment by ryanmrichard** [comment]
@loriab I don't actively use my ORCID (actually I'm not sure I have one...); either way not listing an ORCID for me is fine.

**Comment by xwang862** [comment]
Hi Lori,

My ORCID is 0000-0003-1402-7522. Thanks!

Best wishes,
Xiao Wang

**Comment by PeterKraus** [comment]
0000-0002-4359-5003

Thanks!

**Comment by robashaw** [comment]
0000-0002-9977-0835

Thanks :)

**Comment by hokru** [comment]
0000-0002-0560-1513 Thanks

File: codemeta.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+            "familyName": "Sokolov",
+            "affiliation": "Division of Chemistry and Chemical Engineering, California Institute of Technology",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/ssh2"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Roberto",
+            "familyName": "Di Remigio",
+            "affiliation": "Centre for Theoretical and Computational Chemistry, UiT, The Arctic University of Norway",
```
Comment (robertodr) [inline codemeta.json]: Can the affiliation be changed to "Hylleraas Centre for Quantum Molecular Sciences, UiT - The Arctic University of Norway"?

File: codemeta.json [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+            "familyName": "Sherrill",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0002-5570-7666",
+            "@id": "https://github.com/CDSherrill"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Peter",
+            "familyName": "Kraus",
+            "affiliation": "Liebniz University, Hannover, Germany",
```
Comment (hokru) [inline codemeta.json]: should be _Leibniz_

Comment (loriab) [reply codemeta.json]: got it, thanks

File: codemeta.json [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+            "@id": "https://orcid.org/0000-0002-4359-5003",
+            "@id": "https://github.com/PeterKraus"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Holger",
+            "familyName": "Kruse",
+            "affiliation": "Institute of Biophysics, Czech Republic",
```
Comment (hokru) [inline codemeta.json]: please add the full name, there might be others: _Institute of Biophysics of the CAS, v.v.i., Czech Republic_

Comment (loriab) [reply codemeta.json]: out of curiosity, what's "v.v.i."?

Comment (hokru) [reply codemeta.json]: I believe it stands for 'public research institution' (in czech). Don't ask why the English name has it...

**Pull request merged**